src/AuthModule/Components/Register/Register.tsx

```diff
@@ -1,12 +1,398 @@
+                  <TextField
+                    className={`${styleRegister.textField} `}
+                    hiddenLabel
+                    id="Phone"
+                    placeholder="Enter Your Phone Number"
+                    variant="filled"
+                    type="tel"
+                    {...register("phoneNumber", {
+                      required: "phoneNumber is required",
+                      pattern: {
+                        value: /^01\d{9}$/,
+                        message: "Phone number must start with 01 and be 11 digits in total",
+                      },
+                    })}
+                  />
+                   {errors.phoneNumber && (
+                    <Alert className=" mt-1" severity="error">
+                       {errors.phoneNumber.message?.toString()}
+
+                       
+                       </Alert>
+                 
+                  )}
+                </FormControl>
+
+                <div className=" row">
+                  <div className=" col-md-6">
+                    {/* Country */}
+                    <FormControl className="mt-4" variant="standard">
+                      <label className="mb-2 d-block" htmlFor="Country">
+                        Country
+                      </label>
+                      <TextField
+                        className={`${styleRegister} d-block`}
+                        hiddenLabel
+                        id="Country"
+                        placeholder="Enter Your Country"
+                        variant="filled"
+                        type="text"
+                        {...register("country", {
+                          required: "country is required",
+                        })}
+                      />
+                        {errors.country && (
+                    <Alert className=" mt-1" severity="error">
+                                             {errors.country.message?.toString()}
+
+                      
+                       </Alert>
+                 
+                  )}
+                    </FormControl>
+                  </div>
+                  <div className=" col-md-6">
+                    {/* Email Address */}
+                    <FormControl className="mt-4" variant="standard">
+                      <label className="mb-2 d-block" htmlFor="Email">
+                        Email Address
+                      </label>
+                      <TextField
+                        className={`${styleRegister.textField}`}
+                        hiddenLabel
+                        id="Email"
+                        placeholder="Enter Your Email Address"
+                        variant="filled"
+                        type="email"
+                        {...register("email", {
+                          required: "email is required",
+                          pattern: {
+                            value:
+                              /[A-Za-z0-9._%+-]+@(gmail|yahoo|email)\.com/,
```

Comment:
There are other domains, you need to look for a better regular expression for email validation.